### PR TITLE
numba implementation of histogram computation

### PR DESCRIPTION
**Pull request by guillaumegrolleron**
-add numba implementation of histogram computation
-add documentation in charge.py

### CODE REVIEW

**Review by jlenain — COMMENTED** [review]
Thanks a lot, @guillaumegrolleron !
I am testing the vectorization from `numba` locally now, as we discussed. Will merge quickly after my local test passes.

File: nectarchain/user_scripts/ggrolleron/load_wfs_compute_charge.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import json
 
 import logging
-logging.basicConfig(format='%(asctime)s %(name)s %(levelname)s %(message)s',level=logging.INFO,filename = f"{os.environ.get('NECTARCHAIN_LOG')}/{Path(__file__).stem}_{os.getpid()}.log")
+logging.basicConfig(format='%(asctime)s %(name)s %(levelname)s %(message)s',level=logging.DEBUG,filename = f"{os.environ.get('NECTARCHAIN_LOG')}/{Path(__file__).stem}_{os.getpid()}.log")
```
[the remark this answers is not in the document]
Comment (jlenain) [reply nectarchain/user_scripts/ggrolleron/load_wfs_compute_charge.py:13]: @guillaumegrolleron , how much RAM is needed to run on a couple of FITS.FZ files ? The process gets killed after filling ~25 GB of RAM on my side.

**Comment by jlenain** [comment]
@guillaumegrolleron, how much RAM is needed to run on a couple of FITS.FZ files ? The process gets killed after filling ~25 GB of RAM on my side.

**Comment by jlenain** [comment]
Following #35 , could you please `git mv` everything under `nectarchain` to `src/nectarchain` to avoid merge conflicts ?

File: nectarchain/user_scripts/ggrolleron/load_wfs_compute_charge.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jlenain) [reply nectarchain/user_scripts/ggrolleron/load_wfs_compute_charge.py]: Nevermind actually, you are free to do whatever you want in a user script.

**Comment by guillaumegrolleron** [comment]
Hi @jlenain,
All is done on my side ! you can merge

**Comment by jlenain** [comment]
Thanks a lot, @guillaumegrolleron , I'll yet run another test after the rebase before merging.

**Comment by jlenain** [comment]
All good, merging now. Many thanks, @guillaumegrolleron !

**Pull request merged**